vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -56,27 +56,30 @@ Use `ent help` to review the list of available commands.
 ~~~~~~~~~~~~~~~~~~~
 
 > Essentials:
-  - Activate using:  ~/.entando/ent/quickstart/cli/v6.3.0/activate
-  - Dectivate using: ~/.entando/ent/quickstart/cli/v6.3.0/deactivate
+  - Activate using:  /Users/josephwhite/.entando/ent/v6.3.2/cli/v6.3.2/activate
+  - Dectivate using: /Users/josephwhite/.entando/ent/v6.3.2/cli/v6.3.2/deactivate
 
 > Available commands:
   - app                  => Helps managing an EntandoApp
   - app-info             => Displays information about an entando app
   - bundler              => Wrapper for the ENT private installation of the entando bundle tool.
   - check-env            => Checks the environment for required dependencies and settings
   - diag                 => Runs some diagnostic and collects the related info in a tgz file
+  - ecr                  => Helps managing an the Entando ECR
+  - ent                  => Helps managing an the local ents
   - help                 => Helps in having help
   - host                 => Helps managing the system that hosts the quickstart VM
   - jhipster             => Wrapper for the ENT private installation of jhipster.
   - kubectl              => Helper for using kubectl in ent managed scenarios
   - npm                  => Wrapper for the ENT private installation of npm. This is mostly for internal use
   - pod                  => Displays information related to a set of pods
   - prj                  => Helps managing Entando bundle projects
+  - profile              => Helps managing an EntandoApp
   - quickstart           => Helps locally installing entando instances
   - run-tests            => Run the internal tests
 
 > Further info about entando:
-  - ~/.entando/ent/quickstart/cli/v6.3.0/README.md
+  - ~/.entando/ent/v6.3.2/cli/v6.3.2/README.md
```

Comment:
I think it will still be quickstart once released but maybe @w-caffiero-entando can indicate what we should show here.

Reply:
Fixed